scripts/helper-image.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
```

Comment:
I don't see anywhere that it would matter with the current version of the script, but maybe add `set -eo pipefail` just in case?